packages/cli/lib/discoverManifests.js

```diff
@@ -14,6 +15,15 @@ export default async function * () {
     },
   })
 
+  const dir = process.cwd()
```

Comment:
I would not rely on `cwd`. Maybe use [`pkg-dir`](https://github.com/sindresorhus/pkg-dir) to find the package root

```suggestion
  const dir = await packageDirectory()
```

Reply:
ah, ok. I took the usage of `cwd` from the implementation of `findPlugins`